lib/shellable.ts

```diff
@@ -386,8 +395,8 @@ export class LinuxPlatform extends ShellPlatform {
 export class WindowsPlatform extends ShellPlatform {
   public readonly platformType = PlatformType.Windows;
 
-  public prebuildCommands(assumeRole?: AssumeRole): string[] {
-    if (assumeRole) {
+  public prebuildCommands(assumeRole?: AssumeRole, useRegionalSTSEndpoints?: boolean): string[] {
+    if (assumeRole || useRegionalSTSEndpoints) {
```

Comment:
Woah, what? Why are regional endpoints not supported on Windows?

Reply:
That was my oversight. I removed the variable from the condition and added undescore to the name as it's now unused.